File: cash.py
from cs50 import get_float
dollar = get_float("Change owed: ")
# Check if input number is positive integer
while dollar < 0:
    print("Invalid, please enter again.")
    dollar = get_float("Change owed: ")
# covert dollar to cent
dollar *= 100
coin = 0
# Getting Coin number base in input number value range
while dollar >= 25:
    dollar -= 25
    coin += 1
while dollar >= 10:
    dollar -= 10
    coin += 1
while dollar >= 5:
    dollar -= 5
    coin += 1
while dollar >= 1:
    dollar -= 1
    coin += 1
print(coin)

File: cash.c

#include <cs50.h>
#include <math.h>
#include <stdio.h>


int main(void)
{
    //Prompt user input - Change owed
    float i;
    do
    {
        i = get_float("Change owed: \n");
        
    }
    while (i < 0);
    
    //Removing decimal
    int cent = round(i * 100);
    
    //Coin number starting number
    int coin = 0;
    
    //Getting Coin number base in input number value range
    while (cent >= 25)
    {
        cent = cent - 25;
        coin ++;
    }
    while (cent >= 10)
    {
        cent = cent - 10;
        coin ++; 
    }
    while (cent >= 5)
    {
        cent = cent - 5;
        coin ++; 
    }
    while (cent >= 1)
    {
        cent = cent - 1;
        coin ++; 
    }
    
    //Print out coin number
    printf("%i\n", coin);
}
